eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -166,7 +166,7 @@ jobs:
         displayName: "Install tool dependencies"
 
       - script: |
-          node index.js --verbose --out "$(Build.ArtifactStagingDirectory)/dependencies.html"
+          node index.js --verbose --dump "$(Build.ArtifactStagingDirectory)/data.js" --out "$(Build.ArtifactStagingDirectory)/dependencies.html"
```

Comment:
Should we do this in a similar way to what @chidozieononiwu did for .NET https://github.com/Azure/azure-sdk-for-net/pull/9784?

Reply:
This is being generated inside reports subdirectory in the build artifacts.

Reply:
![image](https://user-images.githubusercontent.com/8968058/73783967-78050100-4749-11ea-8bf0-951cbdb22781.png)

Reply:
It is going into the reports artifact but that is currently creating an artifact based on everything in the ArtifactStagingDirectory the question is when we pull this artifact how do we determine what all is interesting for us to publish the dependency reports? I see other things in there that aren't really part of the dependency stuff.

Reply:
Yes only data.js and dependencies.html are reports for depepndencies. Rest are component governance and linting reports